routers/web/web.go

```diff
@@ -150,6 +151,9 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 		rw.WriteHeader(200)
 	})
 
+	hc := &healthcheck.HealthChecker{}
```

Comment:
why did you implement it the "single tone" way? you dont have to

Reply:
yes, it does not have to.
I only use it to organize the code.
but if you do not like it, I can remove it.
thus, the `cache` and `database` func will conflict with the mod, I'll have to add prefix to them.

Reply:
done and pushed.